examples/pybullet/unittests/userDataTest.py

```diff
@@ -0,0 +1,237 @@
+import unittest
+import pybullet
+
+PLANE_PATH = "plane.urdf"
+ROBOT_PATH = "r2d2.urdf"
+
+class TestUserDataMethods(unittest.TestCase):
+
+	def setUp(self):
+		self.client_id = pybullet.connect(pybullet.SHARED_MEMORY)
```

Comment:
You cannot connect to SHARED_MEMORY without starting a server. This is done in the userData.py example, https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/examples/userData.py Can you check it out?

Reply:
Thanks, starts a server now. Also looks a bit cleaner using the BulletClient helper class, especially the multiclient test case.
Is there a way to start a server without a GUI?

Reply:
Yes, you can use SHARED_MEMORY_SERVER for a non-gui version.